google/cloud/documentai_v1beta2/services/document_understanding_service/__init__.py

```diff
@@ -1,24 +1,20 @@
 # -*- coding: utf-8 -*-
-#
-# Copyright 2020 Google LLC
```

Comment:
Is github just not understanding the shift between these files correctly? Or should we be concerned about the https -> http and 2020 -> 2019 shift?